rotkehlchen/db/dbhandler.py

```diff
@@ -310,7 +310,9 @@ def _run_actions_after_first_connection(self, password: str) -> None:
             if transient_version != ROTKEHLCHEN_TRANSIENT_DB_VERSION:
                 # "upgrade" transient DB
                 tables = list(cursor.execute('select name from sqlite_master where type is "table"'))  # noqa: E501
+                cursor.executescript('PRAGMA foreign_keys = OFF;')
```

Comment:
have you tested this?

Reply:
Yes, with my database and finally got it working